### PR TITLE
merge 2019.06

### PR DESCRIPTION
**Pull request by kkrebs**
[Link to jira](https://mycore.atlassian.net/browse/MCR-).

### CODE REVIEW

File: mycore-viewer/src/main/typescript/modules/base/components/model/MyCoReBasicToolbarModel.ts [hunk 1]
```diff
@@ -101,7 +101,7 @@ namespace mycore.viewer.model {
             this._zoomOutButton = new widgets.toolbar.ToolbarButton('ZoomOutButton', '', 'zoom-out', 'search-minus');
             this._zoomWidthButton = new widgets.toolbar.ToolbarButton('ZoomWidthButton', '', 'zoom-width', 'arrows-alt-h');
             this._zoomFitButton = new widgets.toolbar.ToolbarButton('ZoomFitButton', '', 'zoom-fit-in', 'expand-arrows-alt');
-            this._rotateButton = new widgets.toolbar.ToolbarButton('RotateButton', '', 'Rotate', 'undo');
+            this._rotateButton = new widgets.toolbar.ToolbarButton('RotateButton', '', 'Rotate', 'redo');
```
Comment (adlerre) [inline mycore-viewer/src/main/typescript/modules/base/components/model/MyCoReBasicToolbarModel.ts]: Codacy Issue found: ['widgets' is not defined.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5130243)

**Comment by adlerre** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5130243&bid=16859322)

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on c02175deac2c6decf5002e800a534b7ffaa1ae3d! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5300-pull_request-923

**Comment by adlerre** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5130243&bid=16859322)

File: mycore-viewer/src/main/typescript/modules/base/components/model/MyCoReBasicToolbarModel.ts (hunk 1 above)
Comment (adlerre) [inline mycore-viewer/src/main/typescript/modules/base/components/model/MyCoReBasicToolbarModel.ts]: Codacy Issue found: ['widgets' is not defined.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5130243)

**Comment by adlerre** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5130243&bid=16859322)

File: mycore-viewer/src/main/typescript/modules/base/components/model/MyCoReBasicToolbarModel.ts (hunk 1 above)
Comment (adlerre) [inline mycore-viewer/src/main/typescript/modules/base/components/model/MyCoReBasicToolbarModel.ts:104]: Codacy Issue found: ['widgets' is not defined.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5130243)

**Pull request merged**